examples/README.txt

```diff
@@ -27,3 +27,21 @@ These examples are part of the Protocol Buffers tutorial, located at:
 * Note that on some platforms you may have to edit the Makefile and remove
 "-lpthread" from the linker commands (perhaps replacing it with something else).
 We didn't do this automatically because we wanted to keep the example simple.
+
+## C Sharp ##
+
+The C# sample is provided in its own directory:
+  https://github.com/google/protobuf/tree/master/csharp/src/AddressBook
+
+## Go ##
+
+Go support is provided by a plugin to the protocol buffer compiler:
+  https://github.com/golang/protobuf for more information.
+
+First, install the the Protocol Buffers compiler.  Then, install the Go Protocol
```

Comment:
consider "the Protocol Buffers compiler (protoc)" - you mention `protoc` in the following line, but the user may not know that that is the command name for the protocol buffers compiler.